helm/managed/logging/elastic/values.yaml

```diff
@@ -84,6 +87,21 @@ kibana:
   service:
     type: "LoadBalancer"
     annotations:
+#We want to expose Kibana's UI via link on ECLWATCH...
+#Kibana's default back-end FQDN: http://<releasename>-kibana.default.svc.cluster.local:5601
+#Exact back-end and/or front-end FQDN should be determined using k8s API, not static info in annotations
+      #Required
+      hpcc.eclwatch.io/enabled: "true"
+      #Optional but informative
+      hpcc.eclwatch.io/description: "Provides view in to HPCC component logs"
+      hpcc.eclwatch.io/label : "HPCC Logs via Kibana"
+
+      #Optional if targeting custom Kibana UI (different than service defined in this yaml)
+      #"hpcc.eclwatch.link.io/protocol"
+      #"hpcc.eclwatch.link.io/host"
+      #"hpcc.eclwatch.link.io/port"
+#Routes landing page to Kibana's discover section
+      hpcc.eclwatch.io/path: "/app/discover"
```

Comment:
@rpastrana I think they should all be `hpcc.eclwatch.io` and no `hpcc.eclwatch.link.io`?
@wangkx I would suggest the following ESP Method:
```
hpccAnnotations("eclwatch.io") => [{"enabled": "true", "description": "Provides view in to HPCC component logs", label: "HPCC Logs via Kibana", "path": "/app/discover"}];
```

Reply:
Maybe "hpccAnnotationNamespaces" might be better?